src/screens/claimables/ClaimingTransactionClaimable.tsx

```diff
@@ -157,7 +158,7 @@ export const ClaimingTransactionClaimable = ({ claimable }: { claimable: Transac
 
       const { errorMessage } = await walletExecuteRap(wallet, {
```

Comment:
am i lost or this is using current raps logic

i was just reviewing thi https://github.com/rainbow-me/rainbow/pull/6138

Reply:
it uses v2